### PR TITLE
Add description field to file properties modal

### PR DESCRIPTION
**Pull request by dinomut1**
Adds text area field for editing a file's description

### CODE REVIEW

File: packages/ui/src/primitives/tailwind/TextArea/index.tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+      outlined: 'gap-2',
+      underlined: '',
+      onboarding: ''
+    }
+
+    const twcontainerClassName = twMerge(
+      'flex w-full flex-col items-center',
+      containerVariants[variant],
+      containerClassName
+    )
```
Comment (Rezmason) [inline packages/ui/src/primitives/tailwind/TextArea/index.tsx:98]: Is this preferable to `flex w-full flex-col items-center ${containerVariants[variant] ${containerClassName}`? What's the difference?

**Review by Rezmason — APPROVED** [review]

File: packages/ui/src/primitives/tailwind/TextArea/index.tsx
Comment (aditya-mitra) [inline packages/ui/src/primitives/tailwind/TextArea/index.tsx:1]: have asked Yulia for a textarea input design.
it it best to use a design system currently.

**Review by aditya-mitra — CHANGES_REQUESTED** [review]
waiting for text area input
(I will add the text area design in this PR itself)

**Comment by dinomut1** [comment]
We need this to proceed with using this field for wizard development. We can swap this design out for whatever Yulia decides upon later, but for now i'm going to go ahead and merge this in

**Pull request merged**